### PR TITLE
Compatibility with the next version of ACI

### PR DESCRIPTION
**Pull request by davidyuk**
https://github.com/aeternity/aesophia/pull/421

### CODE REVIEW

**Review by dincho — CHANGES_REQUESTED** [review]
- please add a @deprecated notice around the line
- please add a release note that states that ACI compiler version X is deprecated 

In general the old format would be dropped in next major release of the lib

File: src/AciTypeResolver.js [hunk 1]
```diff
@@ -304,7 +304,11 @@ class AciTypeResolver {
         }
 
         const def = [
-            ...namespaceData.type_defs,
+            /**
+             * @deprecated ACIs generated by aesophia@7 and below
```
Comment (davidyuk) [inline src/AciTypeResolver.js:308]: It sounds strange because I can't say what to use instead due to `typedef` change is not released yet

Comment (dincho) [reply src/AciTypeResolver.js:308]: That was my point in the previous comment, once release we can update this PR and merge it.

**Comment by dincho** [comment]
The release note is misleading that only HTTP compiler is supported, please mention that one can/should use CLI ACI (any version)

**Comment by davidyuk** [comment]
The thing is that there is now new release of cli that emits new aci

**Comment by davidyuk** [comment]
aesophia_cli@7.1.0 released, so I've updated the changelog 
https://github.com/aeternity/aesophia_cli/releases/tag/v7.1.0

**Pull request merged**